src/domains/network/components/NetworkIcon/NetworkIcon.tsx

```diff
@@ -61,7 +61,7 @@ export const NetworkIcon: React.VFC<NetworkIconProperties> = ({
 			<Icon
 				className={iconClassName}
 				data-testid="NetworkIcon__icon"
-				name={network.ticker()}
+				name={network.coinName() === "Mainsail" ? "Mainsail" : network.ticker()}
```

Comment:
I'd move this to a helper file to avoid a lot of string mainsail comparisons in the codebase, e.g. like https://github.com/ArdentHQ/arkvault/pull/436/files#diff-c190d8bc8a7f39c14512908fff436458505b9ba228bcd5eecbb24a27eed0ac73R172